fixtures/node_annotate.py

```diff
@@ -0,0 +1,115 @@
+import py
+import pytest
+import csv
+import yaml
+import os
+from operator import itemgetter
+
+ROOT = py.path.local(__file__).dirpath().dirpath()
+
+
+class MarkFromMap(object):
+    def __init__(self, mark_map):
+        self.mark_map = mark_map
+
+    def pytest_itemcollected(self, item):
+        mark = self.mark_map.get(item.nodeid)
+        if mark is not None:
+            item.add_marker(mark)
+
+    @classmethod
+    def from_parsed_list(cls, parsed, key, map_value):
+        data = dict(map(itemgetter('id', key), parsed))
+        mark_map = dict((k, map_value(v)) for k, v in data.items())
+        return cls(mark_map)
+
+
+def pytest_configure(config):
+    path = config.getini('cfme_annotations_path')
```

Comment:
We would probably add this to one of the yamls that we have, so that we can use the utils.conf module

```python 
from conf import cfme_data
cfme_data['cfme_annotations_path']
```

As an example...